src/sonic-config-engine/minigraph.py

```diff
@@ -645,27 +649,63 @@ def parse_spine_chassis_fe(results, vni, lo_intfs, phyport_intfs, pc_intfs, pc_m
 #
 ###############################################################################
 
-def filter_acl_mirror_table_bindings(acls, neighbors, port_channels):
+def filter_acl_table_bindings(acls, neighbors, port_channels, sub_role):
+    filter_acls = {}
     """
-        Filters out inactive front-panel ports from the binding list for mirror
-        ACL tables. We define an "active" port as one that is a member of a
-        port channel or one that is connected to a neighboring device.
+        If the asic role is BackEnd no ACL Table (Ctrl/Data/Everflow) is binded.
+        This will be applicable in Multi-NPU Platforms.
     """
+    if sub_role == 'BackEnd':
+        return filter_acls
+
+    front_port_channel_intf = []
+   
+    # Get the front panel port channel. It will use port_alias_asic_map
+    # which will get populated from port_config.ini for Multi-NPU 
+    # architecture
+    for port_channel_intf in port_channels:
+        front_port_channel = True
+        for port_channel_member in port_channels[port_channel_intf]['members']:
+            if port_channel_member in port_alias_asic_map and port_channel_member.startswith(backend_asic_interface_name_prefix):
+                front_port_channel = False
+                break
+        if front_port_channel == True:
+            front_port_channel_intf.append(port_channel_intf)
 
     for acl_table, group_params in acls.iteritems():
         group_type = group_params.get('type', None)
+        filter_acls[acl_table] = acls[acl_table]
 
-        if group_type != 'MIRROR' and group_type != 'MIRRORV6':
+        if group_type == 'CTRLPLANE':
             continue
 
-        active_ports = [ port for port in group_params.get('ports', []) if port in neighbors.keys() or port in port_channels ]
-
+        # Filters out back-panel ports from the binding list for Data/Everflow
+        # ACL tables. We define an "back-panel" port as one that is a member of a
+        # port channel connected to back asic or directly connected to back asic.
+        # This will be applicable in Multi-NPU Platforms.
+        front_panel_ports = []
+        for port in group_params.get('ports', []):
+            if port in port_alias_asic_map and port.startswith(backend_asic_interface_name_prefix):
+                continue
+            if port in port_channels and port not in front_port_channel_intf:
+               continue
```

Comment:
Indent is not consistent

Reply:
@ Done

---
In reply to: [421178662](https://github.com/Azure/sonic-buildimage/pull/4491#discussion_r421178662) [](ancestors = 421178662)

Reply:
Indent still looks incorrect.

Reply:
@ Done

---
In reply to: [421713114](https://github.com/Azure/sonic-buildimage/pull/4491#discussion_r421713114) [](ancestors = 421713114)